tasks/webpack-dev-server.js

```diff
@@ -45,7 +43,10 @@ yarn add webpack-dev-server`);
       const devClient = [
         `webpack-dev-server/client?${protocol}://${opts.host}:${opts.port}`
       ];
-      if (opts.hot) devClient.push('webpack/hot/dev-server');
+      if (opts.hot) {
+        devClient.push('webpack/hot/dev-server');
+        webpackOptions.plugins.plush(new webpack.HotModuleReplacementPlugin());
```

Comment:
Should this be plush or push? Just noticed this, not sure if plush is a thing. 
.